package-lock.json

```diff
@@ -16234,8 +16207,37 @@
         "url": "https://www.patreon.com/infusion"
       }
     },
+    "node_modules/framer-motion": {
```

Comment:
ah yeah here it is. pulling in master should also get rid of this from the diff and explains that translateZ update to the snapshot